status/mtls_status.go

```diff
@@ -0,0 +1,26 @@
+package status
+
+import (
+	"github.com/kiali/kiali/business"
+)
+
+func (si *StatusInfo) getmTLSStatus() {
+	// Get business layer
+	business, err := business.Get()
+	if err != nil {
+		Put(ClusterMTLS, "error")
+		return
+	}
+
+	isGlobalmTLSEnabled, err := business.IstioConfig.IsMTLSGloballyEnabled()
+	if err != nil {
+		Put(ClusterMTLS, "error")
+	}
+
+	status := "Not globally enabled"
```

Comment:
I wonder if it's better to return here a true/false (or a list of values) instead than a literal.
Otherwise UI will need to parse and check return === "Not globally enabled".

Reply:
Returning "NOT_GLOBALLY_ENABLED", "GLOBALLY_ENABLED". We can extended with new scenarios that seems that could come in a near future.